packages/odyssey-react/src/components/Button/Button.stories.tsx

```diff
@@ -10,8 +10,10 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { Story } from "@storybook/react";
 import React from "react";
+import type { ReactText } from "react";
+import { Story } from "@storybook/react";
+import { useIntl } from "react-intl";
```

Comment:
Note: intentionally making use of react-intl's [imperative API](https://formatjs.io/docs/react-intl/api/), I was using their component initially but realized it would force me to update component type in certain instances.